Tweaks to chart format and colors

nathan-weinberg opened the pull request:
## Motivation

This PR makes some stylistic suggestions to the chart formatting and coloring as well as some minor tweaks to the Bug Template and Healthcheck action

## Changes
![image](https://github.com/transitmatters/t-performance-dash/assets/31703736/165bc32a-48dc-48bb-a812-e062f5b0cf40)

- Clarified that `Version` in Bug Template referred to `Browser Version`
- Users can now trigger `healthcheck` action on demand
- Updated Historic Maximums to include commas to match labels on vertical axis
- Changed `trips` to `Trips` in Service graph to match other graphs' vertical axis
- Changed `Trips` in Ridership graph to `Riders`
- Added descriptor of Historic Maximum to all graphs for readability and consistency with the horizontal axis
- Made `MPH` box background more white than gray to match the graph background

## Testing Instructions

Deploy this PR locally

nathan-weinberg commented on modules/service/ServiceGraph.tsx:
```diff
@@ -115,9 +115,8 @@ export const ServiceGraph: React.FC<ServiceGraphProps> = ({
-                      return `${context.datasetIndex === 0 ? 'Actual:' : 'Scheduled:'} ${
-                        context.parsed.y
-                      } (${((100 * context.parsed.y) / peak).toFixed(1)}% of historical maximum)`;
+                      return `${context.datasetIndex === 0 ? 'Actual:' : 'Scheduled:'} ${context.parsed.y
+                        } (${((100 * context.parsed.y) / peak).toFixed(1)}% of historical maximum)`;
```
For the record my VSCode did this automatically and wouldn't let me save without making this change

devinmatte replied:
@nathan-weinberg Check your locally installed package versions. This to me seems like you may have an older version of prettier locally

nathan-weinberg replied:
@devinmatte I've got `9.6.7`

devinmatte replied:
Huh? `prettier`'s latest version is `3.0.1` 😕

nathan-weinberg replied:
My bad, it's `3.0.0`

nathan-weinberg replied:
Linting fixed, time travel work will be continued in the next PR :wink:

PatrickCleary commented:
I would like to look at this more in-depth before it is merged. Just commenting so I don't forget.

PatrickCleary commented on modules/ridership/RidershipGraph.tsx:
```diff
@@ -71,7 +71,9 @@ export const RidershipGraph: React.FC<RidershipGraphProps> = ({
-                  label: `Historical Maximum (${PEAK_RIDERSHIP[routeIndex ?? 'DEFAULT']})`,
+                  label: `Historical Maximum (${PEAK_RIDERSHIP[
+                    routeIndex ?? 'DEFAULT'
+                  ].toLocaleString('en-us')} riders)`,
```
I like this change. Do we want to also format as `213.7k riders` like in the widget?
<img width="1109" alt="Screenshot 2023-08-18 at 1 13 01 PM" src="https://github.com/transitmatters/t-performance-dash/assets/46229349/718e9597-2b3d-4ce8-a768-1072d426e4df">

nathan-weinberg replied:
Honestly I kinda like the `k` formatting but I'm open to changing it!

PatrickCleary replied:
Yea, I'm saying have them both use `K`

nathan-weinberg replied:
Hmmm I feel like the graph/axis it may make more sense to keep the exact number since that data is a bit more in-depth/granular than the overview, wdyt?

PatrickCleary replied:
Yea, that's a good point. Works for me.

austinjpaul replied:
Can we add a "daily" or "weekly" label to It? In reading it, I can't figure out what the numbers mean. I know it's averaged weekly, but is it avg per day or total per week?

nathan-weinberg replied:
Would it not be the weekday average for the selected timeframe?

PatrickCleary reviewed (approved):
Thanks for doing this clean up.

Looks good to me. WRT the conversaation yesterday – I am a fan of using `taps`/`validations`/`card taps` for the unit of ridership over `riders`

But not too opinionated on which alternative to use.

nathan-weinberg commented:
Made a few more changes based on feedback:
- Changed some "rider" references to "fare validations"
- Specified service is measured by **round** trips
- Added comma formatting to ridership widget value

mathcolo commented on common/types/basicWidgets.tsx:
```diff
@@ -188,7 +188,7 @@ export class MPHWidgetValue extends BaseWidgetValue implements WidgetValueInterf
 export class RidersWidgetValue extends BaseWidgetValue implements WidgetValueInterface {
   getUnits() {
-    return 'Riders';
+    return 'Fare Validations';
```
Same with the `V`?

mathcolo commented on common/types/basicWidgets.tsx:
```diff
@@ -145,7 +145,7 @@ export class PercentageWidgetValue extends BaseWidgetValue implements WidgetValu
 export class TripsWidgetValue extends BaseWidgetValue implements WidgetValueInterface {
   getUnits() {
-    return 'Trips';
+    return 'Round Trips';
```
Lowercase the `T` to match other usages in your PR?

nathan-weinberg replied:
I [still think](https://github.com/transitmatters/t-performance-dash/pull/692#issuecomment-1601704806) this capitalization scheme is less appealing to the eye but I will change it for consistency :wink:

mathcolo commented on modules/service/ServiceGraph.tsx:
```diff
@@ -73,7 +73,7 @@ export const ServiceGraph: React.FC<ServiceGraphProps> = ({
                 },
                 {
-                  label: `MBTA scheduled trips`,
+                  label: `Scheduled round trips`,
```
Yessss like this
